### PR TITLE
v3.1.1 CHANGELOG

### PR DESCRIPTION
**Pull request by pohly**
**What type of PR is this?**
/kind documentation

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

File: CHANGELOG/CHANGELOG-3.1.md [hunk 1]
```diff
@@ -1,7 +1,25 @@
...
+# Release notes for v3.1.1
 
 [Documentation](https://kubernetes-csi.github.io)
 
+# Changelog since v3.1.0
+
+## Changes by Kind
+
+### Bug or Regression
+ - Storage capacity: managed-by label potentially too long with a long node name and enableNodeDeployment set to true ([#739](https://github.com/kubernetes-csi/external-provisioner/pull/739), [@akalenyu](https://github.com/akalenyu))
```
Comment (jingxu97) [inline CHANGELOG/CHANGELOG-3.1.md:10]: so this is the only critical bug we want to fix in a patch release? Just want to understand how we pick which one to include or not.

**Comment by humblec** [comment]
@pohly there looks to be some issue with the changelog capture atleast. That said, there are important features/bug fixes since 3.1.0.  Atleast I can think of some the PRs which I have contributed, for ex:

#https://github.com/kubernetes-csi/external-provisioner/pull/713

then there are many other PRs which got release notes in it
ex:
https://github.com/kubernetes-csi/external-provisioner/pull/710
https://github.com/kubernetes-csi/external-provisioner/pull/745
#https://github.com/kubernetes-csi/external-provisioner/pull/733

File: CHANGELOG/CHANGELOG-3.1.md (hunk 1 above)
Comment (pohly) [reply CHANGELOG/CHANGELOG-3.1.md:10]: A user asked for it. There is no systematic triaging of bugs and no automatic backporting of bugs. 

From https://kubernetes-csi.github.io/docs/project-policies.html (emphasis mine):
> Every minor release branch will be supported with patch releases on an *as-needed* basis for at least 1 year

**Comment by pohly** [comment]
> are we planning 4.0.0 with all other enhancements soon ?

There's going to be [a 3.2.0](https://github.com/kubernetes-csi/external-provisioner/pull/746) from the code in master.

> and this meant for only one bug fix available patch release?

Yes.

**Comment by pohly** [comment]
/assign @jsafrane 

For approval.

**Comment by pohly** [comment]
/hold

Let's backport all bug fixes from 3.2.0:
- PR #751 
- PR #750 
- PR #749

**Comment by pohly** [comment]
/hold cancel

Additional backport PRs were merged, changelog was updated.

@jingxu97 : please re-add LGTM

**Comment by pohly** [comment]
@jsafrane: can you also LGTM?

**Pull request merged**